reactos/dll/win32/browseui/travellog.cpp

```diff
@@ -172,6 +172,7 @@ HRESULT STDMETHODCALLTYPE CTravelEntry::Update(IUnknown *punk, BOOL fIsLocalAnch
     GetToolTipText(punk, wch);
     TRACE("Updating entry with display name: %S\n", wch);
 
+    ZeroMemory(&windowData, sizeof(WINDOWDATA));
```

Comment:
So this is something you experience without the rest of your patch applied?  I've never noticed any such behavior (and I generally run with MSVC's uninitialized variable runtime checks). I also only see two GetWindowData implementations in our code. mshtml's always fails, and CShellBrowser's fills all fields.
Do you happen to have a backtrace available for this issue?